src/Altinn.Notifications.Core/Models/Creator.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Altinn.Notifications.Core.Models;
```

Comment:
We're inconsistent with the use of file-scoped namespaces. We can add an .editorconfig file where we turn it on.

Reply:
Updated .editorconfig with file_scoped:suggestion for namespace declarations